src/coreComponents/physicsSolvers/inducedSeismicity/ExplicitQDRateAndState.cpp

```diff
@@ -455,24 +280,29 @@ real64 QuasiDynamicEQRK32::setNextDt( real64 const & currentDt, DomainPartition
   } );
 
   // Compute update factor to currentDt using PID error controller + limiter
-  real64 const dtFactor = m_controller.computeUpdateFactor( m_butcherTable.algHighOrder, m_butcherTable.algLowOrder );
-  real64 const nextDt = dtFactor*currentDt;
+  m_stepUpdateFactor = m_controller.computeUpdateFactor( m_butcherTable.algHighOrder, m_butcherTable.algLowOrder );
```

Comment:
this controller is also something new and not sure if it is really needed? can it be just part of the solver?